napari_allencell_annotator/view/main_view.py

```diff
@@ -157,6 +156,10 @@ def _csv_json_import_selected_evt(self, file_path: Path):
                 # start at row 0 if annotation data was not used from csv
                 file.close()
             self._annotator_model.set_all_images(image_list)
+            if shuffled:
+                self._annotator_model.set_shuffled_images(
+                    FileUtils.shuffle_file_list(self._annotator_model.get_all_images())
+                )
```

Comment:
if we have an else statement here to clear the shuffled image list if there is one, it would fix the issue where the file widget doesnt change if we load in a csv while it is shuffled.

Reply:
added in https://github.com/aics-int/napari-allencell-annotator/pull/70/commits/f332e17ed072e22bcc12976cda03d1e2f7282abd.